manifests/k/KDE/Elisa/23.04.00/KDE.Elisa.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: KDE.Elisa
+PackageVersion: 23.04.00
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Elisa_Release_win64/lastSuccessfulBuild/artifact/elisa-23.04.0-1488-windows-cl-msvc2019-x86_64.exe

```

Comment:
```suggestion
  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Elisa_Release_win64/lastSuccessfulBuild/artifact/elisa-23.04.0-1489-windows-cl-msvc2019-x86_64.exe
```

Reply:
It appears that their build number incremented.

Reply:
@stephengillie Yeah, and this is going to be a problem in long run. I am not sure about the frequency of the update. Shall I include the build number as well in the manifest?